shell/edit/auth/saml.vue

```diff
@@ -99,6 +119,23 @@ export default {
         // Use a spread of config, so that if don't make changes to the defaults if the user edits them
         this.$set(this.model, 'openLdapConfig', { ...LDAP_DEFAULTS });
       }
+    },
+    // sloType is defined on shell/mixins/auth-config.js
+    sloType(neu) {
+      switch (neu) {
+      case SLO_OPTION_VALUES.rancher:
+        this.$set(this.model, 'logoutAllEnabled', false);
```

Comment:
NOTE: After the Vue3 migration, we'll want to come back and replace `this.$set()` with assignment[^1]. 

```suggestion
        this.model.logoutAllEnabled = false;
```

[^1]: https://v3-migration.vuejs.org/breaking-changes/#removed-apis